src/WriteStream.h

```diff
@@ -14,10 +14,10 @@
 
 #pragma once
 
-#include "RWOperation.h"
 #include "RamTypes.h"
 #include "RecordTable.h"
 #include "SymbolTable.h"
+#include "Util.h"
```

Comment:
What is Util.h used for? It includes most system library headers so it's a heavy dependency to have.
std::map is used now, though, and should be added.